test/functional/kernel/hyperplane/tests/test-kernel-hyperplane-2D.hpp

```diff
@@ -11,43 +11,64 @@
 #include <numeric>
 #include <type_traits>
 
-template <typename INDEX_TYPE, typename DATA_TYPE, typename WORKING_RES, typename EXEC_POLICY, typename REDUCE_POLICY>
-void KernelHyperplane2DTestImpl(const int groups, const int idim, const int jdim)
+template <typename INDEX_TYPE, typename DATA_TYPE, typename EXEC_POLICY, typename REDUCE_POLICY, typename USE_PARAM_REDUCER>
+std::enable_if_t<USE_PARAM_REDUCER::value>
+CallKernel(RAJA::View<DATA_TYPE, RAJA::Layout<3, INDEX_TYPE>>& WorkView,
+           const int idim,
+           const int jdim,
+           const int groups,
+           DATA_TYPE& trip_count,
+           DATA_TYPE& oob_count)
```

Comment:
Whether we put the reduce values first or last in the CallKernel call is not consistent.